src/Application/GameConfig.h

```diff
@@ -10,6 +13,7 @@
 #include "Library/Config/Config.h"
 #include "Engine/Random/RandomEnums.h"
 #include "Library/Logger/LogEnums.h"
+#include "Utility/Format.h"
```

Comment:
I think Format.h is only needed inside the cpp file?

Reply:
It's a refuse. Initially I had the constructor implemented in the header but then I moved it :) Nice catch!